.github/workflows/brief.yml

```diff
@@ -81,6 +81,7 @@ jobs:
         run: |
           sudo apt-get install -y check ccache
           echo PYTHON_LINKING_OPTION="-DPYTHON_USE_DYNAMIC_LOOKUP=ON" >> "${GITHUB_ENV}"
+          echo R_BINDINGS="-DWITH_R=True"
```

Comment:
i think you mean to pipe this into the environment too, right?

Reply:
Yep, thanks!